source/howto/CreateVMFromCommandLine.rst

```diff
@@ -18,100 +18,41 @@ Setting up the environment to select project
 
 After logging into a host, you have to set your environment variables.
 
-You can download a script to set these up for you from: https://openstack.stfc.ac.uk/project/api_access/openrc/. You will need to be signed in to the project you wish to use in the OpenStack web interface for this to work.
+Please download the openrc script to set these up for you, from: https://openstack.stfc.ac.uk/project/api_access/openrc/ 
 
-Or set them yourself in a script:-
+You will need to be signed in to the project you wish to use in the OpenStack web interface for this to work.
+
+####################################
+Finding images
+####################################
+A full detailed list of images is available on our Confluence: `Image Types
+<https://stfc.atlassian.net/l/cp/KQ01NgEr/>`_.
```

Comment:
This URL doesn't seem to work

Reply:
Odd seems fine for me, what method are you viewing with?

Reply:
I tried copying and pasting the URL from the rst file directly which didn't work. I just tried with the preview mode and clicked the link and it worked 🤔

Reply:
Hmm strange, copying and pasting `https://stfc.atlassian.net/l/cp/KQ01NgEr/` works for me as well :sweat_smile:

Reply:
Ah it works for me now as well 😅